pl/cheatsheet-deep-learning.md

```diff
@@ -0,0 +1,321 @@
+
+&#10230; Step 3: Z wykorzystaniem propagacji wstecznej użyj straty aby uzyskać gradient.
+
+<br>
+
+**18. Step 4: Use the gradients to update the weights of the network.**
+
+&#10230; Krok 4: Wykorzystaj gradient aby zaktualizować wagi w sieci neuronowej.
+
+<br>
+
+**19. Dropout ― Dropout is a technique meant at preventing overfitting the training data by dropping out units in a neural network. In practice, neurons are either dropped with probability p or kept with probability 1−p**
+
+&#10230; Dropout ― Dropout jest techniką zapobiegania nadmiernemu dopasowaniu (overfitting) do danych treningowych poprzez pomijanie niektórych neuronów w sieci. W praktyce, neurony są pomijane z prawdopodobieństwem p lub nie są pomijane z prawdopodobieństwem 1-p
+
+<br>
+
+**20. Convolutional Neural Networks**
+
+&#10230; Konwolucyjne Sieci Neuronowe
+
+<br>
+
+**21. Convolutional layer requirement ― By noting W the input volume size, F the size of the convolutional layer neurons, P the amount of zero padding, then the number of neurons N that fit in a given volume is such that:**
+
+&#10230; Wymagania warstwy konwolucyjnej ― Zauwżając, że W to rozmiar danych wejściowych, F to rozmiar neuronów warstwy konwolucyjnej, P rozmiar uzupełnienia zerami, to wymaganą ilość neuronów określamy następująco:
+
+<br>
+
+**22. Batch normalization ― It is a step of hyperparameter γ,β that normalizes the batch {xi}. By noting μB,σ2B the mean and variance of that we want to correct to the batch, it is done as follows:**
+
+&#10230; Normalizacja pakietu (Batch normalization) ― Jest to krok w którym hiperparametry γ,β są wykorzystywane do normalizacji pakietu {xi}. Zauważając, że μB to średnia, a σ2B to wariancja, to normalizacja pakiet wygląda następująca:
+
+<br>
+
+**23. It is usually done after a fully connected/convolutional layer and before a non-linearity layer and aims at allowing higher learning rates and reducing the strong dependence on initialization.**
+
+&#10230; Jest ona zazwyczaj stosowana po warstwie pełnej lub konwolucyjnej, a przed zastosowaniem nieliniowej funkcji aktywacyjnej i ma na celu umożliwienie stosowania dużego współczynnika uczenia i zmniejszenia zależności od inicjalizacji.
+
+<br>
+
+**24. Recurrent Neural Networks**
+
+&#10230; Rekurencyjne Sieci Neuronowe
+
+<br>
+
+**25. Types of gates ― Here are the different types of gates that we encounter in a typical recurrent neural network:**
+
+&#10230; Rodzaje bramek ― Przedstawiamy różne rodzaje bramek, które możemy spotkać w typowych sieciach rekurencyjnych (RNN):
+
+<br>
+
+**26. [Input gate, forget gate, gate, output gate]**
+
+&#10230; [Bramka wejściowa, bramka zapominajca, bramka, bramka wyjściowa]
+
+<br>
+
+**27. [Write to cell or not?, Erase a cell or not?, How much to write to cell?, How much to reveal cell?]**
+
+&#10230; [Pisać do komórki, czy nie?, Wyczyścić komówke, czy nie?, Jak dużo zapisać do komórki?, Jak dużo ujawnić komórce?]
```

Comment:
159 should be without redundant punctuation marks, eg. (...) czy nie? Wyczyścić (...), not "czy nie?, Wyczyścić" ["," after "?"] etc.